concepts/60 Themes and Styles/08 ThemeBuilder/01 Get Started/3 Create a New Theme.md

```diff
@@ -6,4 +6,5 @@ New themes are based on [predefined DevExtreme themes](/concepts/60%20Themes%20a
 
 3. Select a color scheme and change basic appearance settings.
 
-4. If the appearance on the preview meets you requirements, click **Export** to [export the theme](/concepts/60%20Themes%20and%20Styles/08%20ThemeBuilder/40%20Export%20the%20Resulting%20Theme.md '/Documentation/Guide/Themes_and_Styles/ThemeBuilder/#Export_the_Resulting_Theme'). Otherwise, click **Advanced Settings** to navigate to the main view for [additional customization](/concepts/60%20Themes%20and%20Styles/08%20ThemeBuilder/10%20Customize%20the%20Theme.md '/Documentation/Guide/Themes_and_Styles/ThemeBuilder/#Customize_the_Theme').
+4. If the appearance on the preview meets you requirements, click **Export** to [export the theme](/concepts/60%20Themes%20and%20Styles/08%20ThemeBuilder/40%20Export%20the%20Resulting%20Theme.md '/Documentation/Guide/Themes_and_Styles/ThemeBuilder/#Export_the_Resulting_Theme') to a stylesheet. We also recommend that you export your theme's metadata. You will need it to update your custom theme if you decide to migrate to a future DevExtreme version.       
+If the basic appearance settings are not enough, click **Advanced Settings**, and you will navigate to the main view for [additional customization](/concepts/60%20Themes%20and%20Styles/08%20ThemeBuilder/10%20Customize%20the%20Theme.md '/Documentation/Guide/Themes_and_Styles/ThemeBuilder/#Customize_the_Theme').
```

Comment:
- "are not enough" This is the same as the comment regarding "meets you requirements". Can you quantify 'enough'? What is not enough? 
- I'm not sure why you need the future tense here either